RPIPs/RPIP-31.md

```diff
@@ -28,18 +28,45 @@ https://dao.rocketpool.net/t/withdrawal-address-splitter-contract/563
 
 ## Specification
 - The withdrawal address that has existed to date SHALL be known as the primary withdrawal address
+
+### Defaults
+- The RPL withdrawal address SHALL be initialized as unset for new node operators
+- The RPL withdrawal address SHALL be initialized as unset for existing node operators
+  
+### Rewards & Withdrawn RPL
 - If a node's RPL withdrawal address is set, all RPL rewards and withdrawn RPL will go to the RPL withdrawal address
 - If a node's RPL withdrawal address is unset, all RPL rewards and withdrawn RPL will go to the primary withdrawal address
+
+### Setting the Address
 - As a node operator, I MUST be able to set an RPL withdrawal address so that I can direct my RPL bond/rewards
   - If a node's RPL withdrawal address is unset, the change MUST come from the node's primary withdrawal address
   - If a node's RPL withdrawal address is set, the change MUST come from the current RPL withdrawal address
 - As a node operator, I MUST be able to confirm the RPL withdrawal address from the new target address before it takes effect, to ensure it is the correct address
 - As the owner of the RPL withdrawal address, I MUST be able to unset my RPL withdrawal address
+
+### Withdrawing Excess RPL
```

Comment:
I'm not sure "excess RPL" is a clear enough concept. I might just have this be "Withdrawing RPL" and then "MUST be able to withdraw RPL as allowed by the protocol" or smth